### PR TITLE
Add target to run docs without api docs.

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

**Review by gkaemmer — CHANGES_REQUESTED** [review]
Let's get rid of the package-lock, unless you added it for a reason. Other than that looks good.

Maybe also add a `dev` task? 😃
```
dev: install
  HUGO_PURGECSS=false hugo server
```

File: docs/Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 run: api_docs install
 	hugo server
```
Comment (gj) [inline docs/Makefile:2]: nit

```suggestion
run: api_docs
	$(MAKE) run_without_api_docs
```

**Review by gj — APPROVED** [review]
Don't particularly care whether or not these lockfiles remain version-controlled, but we should probably standardize on Yarn or NPM project-wide. I think we use Yarn in most places.

**Comment by dhatch** [comment]
> should probably standardize on Yarn or NPM project-wide. I think we use Yarn in most places.

@gj Is using Yarn vs. NPM the reason the lock file version changed?

**Review by gkaemmer — APPROVED** [review]

**Comment by gj** [comment]
> > should probably standardize on Yarn or NPM project-wide. I think we use Yarn in most places.
> 
> @gj Is using Yarn vs. NPM the reason the lock file version changed?

nope; using Yarn would've generated a (new) `yarn.lock` file in addition to the existing `package-lock.json` file generated by NPM

**Pull request merged**